contact.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <h1 class="center">Contact</h1>
+    <div class="contact">
+        <div class="contactContainer">
+            <img src="img\Message_fill.svg" alt="">
+            <p> <b>Email :</b> contactPro@oxygen-school.com</p>
+        </div>
+        <div class="contactContainer">
+            <img src="img\Phone_fill.svg" alt="">
+            <h5>Telephone </h5>
+            <span class="phone">
+             <p> <b>Fix :</b>  0150 22 32 23</p>
+             <p> <b>Mobile :</b>0150 22 32 23</p>
+            </span>
+        </div>
+        <div class="contactContainer">
+            <img src="img\Send_fill.svg" alt="">
+            <h5>Localisation : </h5>
+            <p> 68 av philipe augueste 75011 paris</p>
+
+        </div>
+
+
+    </div>
+    <form action="#" method="post">
+        <fieldset>
+            <legend>Nom et prenom</legend>
+            <input type="text" name="" id="">
```

Comment:
if you don't use attributes, please suppress'em

Reply:
yes i will but name i will use it with the  backend ... !!